edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/CbsNode.cs

```diff
@@ -37,7 +38,8 @@ class CbsNode : ICbsNode, IAmqpAuthenticator
         readonly IAuthenticator authenticator;
         readonly string iotHubHostName;
         readonly ICredentialsCache credentialsCache;
-        readonly ConcurrentDictionary<string, CredentialsInfo> clientCredentialsMap = new ConcurrentDictionary<string, CredentialsInfo>();
+        readonly ConcurrentDictionary<string, ClientToken> clientTokens = new ConcurrentDictionary<string, ClientToken>();
+        readonly ConcurrentDictionary<string, byte> authenticatedClients = new ConcurrentDictionary<string, byte>();
```

Comment:
Does this still need to be ConcurrentDictionary?

Reply:
yea still needed for UpdateCbsToken which happens on a 3rd thread.